changes/3444.bugfix.rst

```diff
@@ -0,0 +1 @@
+`Add error handling for Window content setter AttributeErrors`
```

Comment:
There's no need to put quotes around this; it will be injected as ReST content.
```suggestion
Add error handling for Window content setter AttributeErrors
```